dashboard/test/helpers/workshop_survey_results_helper_test.rb

```diff
@@ -2,6 +2,79 @@
 
 class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
   include Pd::WorkshopSurveyResultsHelper
+  include Pd::JotForm::Constants
+
+  setup_all do
+    @workshop = create :pd_workshop, :local_summer_workshop, course: Pd::SharedWorkshopConstants::COURSE_CSP, num_facilitators: 2
+
+    Pd::SurveyQuestion.create(
+      form_id: CDO.jotform_forms['local']['day_0'],
+      questions: [
+        Pd::JotForm::MatrixQuestion.from_jotform_question(
+          {
+            qid: '1',
+            type: TYPE_MATRIX,
+            name: 'sampleMatrix',
+            text: 'How do you feel about these statements?',
+            order: '1',
+            mcolumns: 'Strongly Agree|Agree|Neutral|Disagree|Strongly Disagree',
+            mrows: 'I am excited for {workshopCourse}|I am prepared for {workshopCourse}'
+          }.stringify_keys
+        ),
+        Pd::JotForm::ScaleQuestion.from_jotform_question(
+          {
+            qid: '2',
+            type: TYPE_SCALE,
+            name: 'sampleScale',
+            text: 'Do you like {workshopCourse}?',
+            order: '1',
+            scaleFrom: '1',
+            scaleAmount: '5',
+            fromText: 'Strongly Agree',
+            toText: 'Strongly Disagree'
+          }.stringify_keys
+        ),
+        Pd::JotForm::TextQuestion.from_jotform_question(
+          {
+            qid: '3',
+            type: TYPE_TEXTBOX,
+            name: 'sampleText',
+            text: 'Write some thoughts here',
+            order: '4'
+          }.stringify_keys
+        )
+      ].to_json
+    )
```

Comment:
I don't think we need or want to test the JotForm parsing here. Part of the point of separating those concerns is so this code doesn't need to know or care about any JotForm internals. The JotForm Question parsing is already well unit tested over there. This consumption & display only needs to understand our internal types, and should more simply construct them explicitly like in this [test setup](https://github.com/code-dot-org/code-dot-org/blob/d37236b428091be4aff59ca05b165960ef209755/dashboard/test/lib/pd/jot_form/form_questions_test.rb#L11-L73).